scheduler/test/cook/test/simulator.clj

```diff
@@ -0,0 +1,586 @@
+(ns cook.test.simulator
```

Comment:
It doesn't seem like this should live under `scheduler/test`. Maybe it should go in the same directory with the example files?